src/core/utils/common.tsx

```diff
@@ -43,3 +43,16 @@ export const getUserColor = (emailOrName: string) => {
   }
   return USER_COLORS[hash];
 };
+
+/**
+ * Get the portal container, or create one if it doesn't exist.
+ */
+export const getContainer = (containerId: string) => {
+  let container = document.getElementById(containerId);
+  if (container === null) {
```

Comment:
Better use `==` as it will handle `undefined` and `null`.

Reply:
fixed